geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_vars.yaml_tmpl

```diff
@@ -425,6 +425,14 @@ vars:
 
   lingua_extractor: {}
 
+  global_headers:
+    X-Frame-Options: SAMEORIGIN
+    X-Xss-Protection: 1; mode=block
+    X-Content-Type-Options: nosniff
+    Referrer-Policy: origin
+    Strict-Transport-Security: max-age=31536000; includeSubDomains
```

Comment:
Then you break localhost access when we develop, no?

Is it possible to remove entries from there with a vars.yaml?

Reply:
Ah, you are saved by that note:
```
Note: The Strict-Transport-Security header is ignored by the browser when your site is accessed using HTTP; this is because an attacker may intercept HTTP connections and inject the header or remove it.  When your site is accessed over HTTPS with no certificate errors, the browser knows your site is HTTPS capable and will honor the Strict-Transport-Security header.
```
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security

Reply:
Yes it's possible to remove with the vars.yaml :-)